eq-author-api/schema/tests/comments.test.js

```diff
@@ -372,7 +375,7 @@ describe("comments", () => {
         pageId: createdCalSumPage.id,
       });
 
-      expect(queriedComment.comments[0].replies).toHaveLength(0);
+      expect(queriedComment.page.comments[0].replies).toHaveLength(0);
     });
   });
 });
```

Comment:
Could we assert that a reply was added before we check to see that it has been deleted, please? It'll add more credibility to the test.

Reply:
make sense - so I've updated